head.html

```diff
@@ -1,5 +1,20 @@
 <meta name="viewport" content="width=device-width, initial-scale=1"/>
 <meta data-privacy-proxy-server="https://privacy-proxy-server.usercentrics.eu"/>
+<script>
+    // Fix for screen flicker issue observed with Target
+    (function(g, b, d) {
+        (function(a, c, d) {
+            if (a) {
+                var e = b.createElement("style");
+                e.id = c;
+                e.innerHTML = d;
+                a.appendChild(e)
+            }
+        }
+        )(b.getElementsByTagName("head")[0], "at-body-style", d);
+    }
+    )(window, document, "body {opacity: 1 !important}");
```

Comment:
I hope this plays well with other `opacity` settings. For e.g. if we set the `opacity` somewhere to `0` , you may want to check if this isn't inadvertently overriding that.

Reply:
Doesn't it only apply this to elements with name `at-body-style`?

Reply:
Maybe @hparwani2 can clarify this question, since the patch comes from him 😄